apps/crn-server/src/routes/research-outputs.route.ts

```diff
@@ -45,44 +47,61 @@ export const researchOutputRouteFactory = (
   );
 
   researchOutputRoutes.post('/research-outputs', async (req, res) => {
-    const { body, loggedInUser } = req;
+    const { body, loggedInUser, query } = req;
     const createRequest = validateResearchOutputPostRequestParameters(body);
     validateResearchOutputPostRequestParametersIdentifiers(createRequest);
 
-    if (
-      !loggedInUser ||
-      !hasCreateUpdateResearchOutputPermissions(
-        loggedInUser,
-        createRequest.teams,
-      )
-    ) {
+    const permissions = getUserPermissions(
+      loggedInUser as UserResponse,
+      'teams',
+      createRequest.teams,
+    );
+
+    const options = validateResearchOutputRequestQueryParameters(query);
+    const published = options.published ?? true;
```

Comment:
once again I don't think we should use a past tense here, on the API request. If I want create a resource I can decide to either `publish` it or not

Reply:
fixed here [review: change published to create options](https://github.com/yldio/asap-hub/pull/2667/commits/b072177f672bd5f5e7ce1a2eeea1d168638ed9ce)